localized/ja/license.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
You can probably remove this file, as the repository root already has a license file in it.

Reply:
OK.
I will remove that.